src/app/user.ts

```diff
@@ -49,16 +42,10 @@ export const login = async (userId: string): Promise<loginResponse> => {
 
 export const register = async (user: Partial<User>): Promise<loginResponse> => {
```

Comment:
can we ask the UI to send us a token that will signal that the user has been authenticated by google or something similar? so not every request sent to this route will be processed but only those that have been authentiacted